src/engine/MultiColumnJoin.h

```diff
@@ -62,7 +64,7 @@ class MultiColumnJoin : public Operation {
    *width resultWidth (or be left vector that should have resultWidth entries).
    *This method is made public here for unit testing purposes.
    **/
-  static void computeMultiColumnJoin(
+  void computeMultiColumnJoin(
```

Comment:
I'm sure sonarcloud says that it can be const (as long as you don't give us important details).